ci/scripts/go-executable-build.sh

```
@@ -0,0 +1,33 @@
+  echo "usage: $0 <package-name>"
+  exit 1
+fi
+package_name=$package
+echo "cleaning dist..."
+rm -r $output_path
+mkdir -p $output_path
+platforms=("windows/amd64" "windows/386" "darwin/amd64" "darwin/arm64" "linux/386" "linux/amd64")
+cd $app_dir
+for platform in "${platforms[@]}"
+do
+	platform_split=(${platform//\// })
```

Shellcheck (0.10.0)

[warning] Quote to prevent word splitting/globbing, or split robustly with mapfile or read -a.

(SC2206)